### PR TITLE
--query, --just-reindex, and rubocop cleanup

### PR DESCRIPTION
**Pull request by mccalluc**
@afred, please review.

### CODE REVIEW

File: scripts/lib/null_logger.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-  def initialize(*args)
+  def initialize(*_args)
   end
-  def add(*args, &block)
+
+  def add(*_args, &_block)
   end
-end
\ No newline at end of file
+end
```
Comment (afred) [inline scripts/lib/null_logger.rb]: @mccalluc what is the point of `NullLogger`?

Comment (mccalluc) [inline scripts/lib/null_logger.rb]: @afred : 
```
$ grep NullLogger -r scripts
scripts/lib/null_logger.rb:class NullLogger < Logger
scripts/lib/downloader.rb:    $LOG ||= NullLogger.new
scripts/lib/pb_core_ingester.rb:    $LOG ||= NullLogger.new
```

If these are being called from download_clean_ingest, $LOG will be set, but if it's being called from the unit tests, we actually don't need or want any output to the log. NullLogger does this.

**Comment by mccalluc** [comment]
@afred: Anything else?

**Comment by mccalluc** [comment]
Thanks! (Sorry for the random comment.)

**Pull request merged**